docs/asset-manager-framework.md

```diff
@@ -6,7 +6,19 @@ It handles the necessary integration with WordPress (Ajax endpoints and Backbone
 
 The intention is that media provided by any external source will become a seamless part of your site's media library.
 
-AMF is included and loaded by Altis of the box.
+## Loading AMF
+
+AMF is loaded automatically when using the Global Media Library feature, however if you would like to use the framework without that feature you can conditionally load it in using the function `Altis\Media\load_amf()`:
+
+```php
+add_action( 'plugins_loaded', function () {
+    Altis\Media\load_amf();
```

Comment:
This example isn't very helpful. It shows how you can load AMF, but it doesn't sufficiently explain why you'd want to do this. A better example might add the aforementioned conditional to determine whether AMF should be loaded.